db/postgres.go

```diff
@@ -386,7 +375,7 @@ func (p *pgClient) TodaysLatency() (map[string][]api.Latency, error) {
 			return nil, fmt.Errorf("Empty application public key, in query result line: %s", r)
 		}
 
-		latencyByHour := api.Latency{Time: hourlyTime, Latency: roundFloat(hourlyAverageLatency, 5)}
+		latencyByHour := api.Latency{Time: hourlyTime, Latency: numbers.RoundFloat(hourlyAverageLatency, 5)}
```

Comment:
Please use the standard math library from GO: https://pkg.go.dev/math

Reply:
resolving per our convo on discord